cmd/ekco/cli/generate-haproxy-config.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/replicatedhq/ekco/pkg/internallb"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func GenerateHAProxyConfigCmd(v *viper.Viper) *cobra.Command {
+	primaries := &[]string{}
+
+	cmd := &cobra.Command{
+		Use:   "generate-haproxy-config",
+		Short: "Generate HAProxy config file for the internal load balancer",
+		Args:  cobra.ExactArgs(0),
+		PreRun: func(cmd *cobra.Command, args []string) {
+			v.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			out, err := internallb.GenerateHAProxyConfig(*primaries...)
+			if err != nil {
+				return err
```

Comment:
why not consistent with returning err or log.Fatalf?

Reply:
`change-load-balancer` is the only one that has its output parsed